app/models/manageiq/providers/ibm_terraform/configuration_manager/configured_system.rb

```diff
@@ -1,4 +1,5 @@
 class ManageIQ::Providers::IbmTerraform::ConfigurationManager::ConfiguredSystem < ::ConfiguredSystem
+  include SupportsFeatureMixin
```

Comment:
Note, this should be coming from `ConfiguredSystem`.... ❓

Reply:
@agrare have you seen this?  Is this a problem because of collapsed namespaces?  It should have this feature mixin from [here](https://github.com/ManageIQ/manageiq/blob/3e2f8293e43f2b8d0694574ffc321d2d5db5037a/app/models/configured_system.rb#L3).

Reply:
It should be automatic...I'm very confused

Reply:
Oh this is rails 6 - is Zeitwerk maybe doing something weird?

Reply:
yeah, I'l research it some more.